awsutils/cloudformation.py

```diff
@@ -28,6 +28,12 @@ def create(self, stack_name, template_body):
         print "\n\n\n%s\n\n\nSTACK CREATED" % stack
         return stack
 
+    def delete(self, stack_name):
+        stack = self.conn_cfn.delete_stack(stack_name)
+
+        print "\n\n\n%s\n\n\nSTACK DELETED" % stack
```

Comment:
Two things here:

1) This has only requested the stack to be deleted when this is printed
2) Info output like this shouldn't really be in this module, but in the fabfile (single responsibility principle)